### PR TITLE
Add registration pages text

### PR DESCRIPTION
**Pull request by ssinger**
Adds pages with registration information. 

(We aren't yet ready to open registration, please don't merge until we are)

### CODE REVIEW

File: registration.html.j2 [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+</p>
+<p>
+  All cancellations incur a $30(plus tax) administration fee.
+</p>
+
+<hr>
+
+<h2 id="student">Students</h2>
+<p>
+  We accept more or less all full-time, as well as part-time, enrolments
```
Comment (jkatz) [inline registration.html.j2]: Shouldn't say "more-or-less"

Comment (jkatz) [reply registration.html.j2]: "enrollments"

File: registration.html.j2 [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+Students who register after the student tickets have been sold out will need to buy a full price regular registration
+</p>
+
+<h3>Your Data</h3>
+<p>
+  When registering for the conference, we will ask for personal information
+  such as name and address, but also your dietary requirements.  For
+  information on how this data is stored and processed, see the
+  <a href="https://www.pgevents.ca/about/privacypolicy/">PG Events Canada
+    Privacy Policy</a>.
```
Comment (jkatz) [inline registration.html.j2]:
```suggestion
  <a href="https://www.pgevents.ca/about/privacypolicy/">Slonik Events Canada
    Privacy Policy</a>.
```

File: registration.html.j2 (hunk 1 above)
Comment (ssinger) [reply registration.html.j2]: I've pushed an update

**Pull request merged**